Mathlib/MeasureTheory/MeasurableSpace/Card.lean

```diff
@@ -31,8 +31,7 @@ variable {α : Type u}
 
 open Cardinal Set
 
--- Porting note: fix universe below, not here
```

Comment:
This porting note still applies, correct?

Reply:
Yes, the universe has to be specified separately from the notation.

Reply:
So shouldn't you keep the porting note?

Reply:
Ah, my bad. Must have deleted it out of instinct.